config/locales/models/assignments/en.yml

```diff
@@ -31,11 +31,13 @@ en:
         short_identifier: Short Identifier
         start_time: Start time
         student_form_groups: Students may form their own groups
+        token_end_date: Tokens available until
         token_period: Tokens regenerate after
         token_start_date: Tokens available on
         tokens_per_period: Tokens per group
         unlimited_tokens: Unlimited tokens
         url_submit: Allow students to submit URLs
+        use_assignment_due_date: Leave blank to use assignment due date
```

Comment:
This isn't an attribute of the model, and so doesn't belong in this file. It should go under views (and related to automated tests)